plugins/polio/preparedness/parser.py

```diff
@@ -375,31 +398,40 @@ def get_national_level_preparedness_v2(spread: CachedSpread):
             kv[indicator_key] = value
             print(indicator_key, value, rowcol_to_a1(*rc))
 
-        kv["round"] = RoundNumber.unknown
+        round_name = worksheet.get_a1("C8")  # Temp fix using hard coded cell position to avoid round mismatch errors
+        if round_name == "Tour 1 / Rnd 1":
+            round = RoundNumber.round1
+        elif round_name == "Tour 2 / Rnd 2":
+            round = RoundNumber.round2
+        else:
+            round = RoundNumber.unknown
+
+        kv["round"] = round
         return kv
     raise Exception(
         "Summary of National Level Preparedness or Summary of Regional Level Preparedness was not found in this document"
     )
 
 
-def parse_prepardness_v2(spread: CachedSpread):
+def parse_prepardness_v2(spread: CachedSpread, exclusions=None):
     preparedness_data = {
-        "national": get_national_level_preparedness_v2(spread),
-        **get_regional_level_preparedness_v2(spread),
+        "national": get_national_level_preparedness_v2(spread, exclusions),
+        **get_regional_level_preparedness_v2(spread, exclusions),
         "format": TEMPLATE_VERSION,
     }
     preparedness_data["totals"] = get_preparedness_score(preparedness_data)
     return preparedness_data
 
 
-def get_preparedness(spread: CachedSpread):
+def get_preparedness(spread: CachedSpread, exclusions=None):
+    exclusions = Config.objects.filter(slug=WORKSHEET_EXCLUSION_SLUG).first()
```

Comment:
Shouldn't it be this?
```suggestion
    exclusions = Config.objects.filter(slug=WORKSHEET_EXCLUSION_SLUG).first().content
```